_profilers/matlab/run-and-time.md

```diff
@@ -0,0 +1,149 @@
+{% endfigure %}
+
+### Flame Graph
+
+*If the flame graph section states "Flame graph is not available...", try reducing the scope of the profile as too much data was collected. If you can't see that message or the graph, make sure the flame graph accordion is expanded.*
+
+Flame graphs are a common approach to visualising aggregate function-level profiling data (or if vertically reflected, called an icicle graph). The bottom row of the flame graph is the root of the profile which occupies 100% of the runtime, this is shown to be the "Profile Summary". Each row above this shows the most expensive functions (or files) executed by the previous row, producing a hierarchical visualisation of where time was being spent during execution. Each function will only appear as a child of the function it was called within once, as multiple child-calls to the same function are aggregated. As rows progress up the graph the proportion of the graph's width occupied by cells typically reduces, producing the flame (or icicle) pattern the graph is named after. These gaps represent the execution time of the function that was not spent executing child functions, e.g., basic arithmetic and control flow.
+
+If a particular cell of MATLAB's flame graph is hovered with the mouse cursor, a tool tip appears saying the percentage runtime (equivalent to the cell's width) and absolute runtime of that function including child function calls. Grey cells represent MATLAB functions, and blue cells represent your code.
+
+In [Figure 2 (above)](#profile-summary) it's clear that `moving_matrix_plotter_limits` was the most expensive function (98.5% of the runtime), that's because we profiled `moving_matrix_plotter_limits.m`. The row above show that `fzero` and `hold` within `moving_matrix_plotter_limits.m` occupied 25.9% and 46.6% of the runtime respectively. That would suggest that those functions, and how are they are being used, should be investigated.
+
+If a cell within the flame graph is clicked, it will navigate to a profile summary for the selected cell's file or function.
+
+### Profile Summary Table
+
+The table beneath the flame graph lists each of the functions called in order of their total execution time. This provides the same information as the flame graph, however it includes all functions called including very quick ones which would be difficult to visualise in the flame graph.
+
+The table has 5 columns:
+
+- **Function Name**, The name of the relevant function or file.
+- **Calls**, The number of times that function (or file) was called.
+- **Total Time**, The absolute time spent executing that function, including child function calls.
+- **Self Time**, The absolute time spent executing that function, excluding child function calls.
+- **Total Time Plot**, A bar representing the total time and self time, with respect to the total runtime. This is how it would appear within the flame graph as a child of the root.
```

Comment:
```suggestion
- **Total Time Plot**, A bar representing the total time and self time, with respect to the total runtime of the profile. This is how it would appear within the flame graph as a child of the root.
```